src/views/selectors/index.ts

```diff
@@ -22,3 +23,21 @@ export const getViewsForDashboard = (
 
   return views
 }
+
+export const getTimeFormatForView = (
+  state: AppState,
+  viewID: string
+): string => {
+  const view = getByID<View>(state, ResourceType.Views, viewID)
+
+  let timeFormat = ''
+  if ('timeFormat' in view.properties) {
+    timeFormat = view.properties.timeFormat
+  }
+
+  if (timeFormat === '') {
+    return DEFAULT_TIME_FORMAT
+  }
+
+  return timeFormat
```

Comment:
we can make this a little more compact if you like:
```suggestion
  if ('timeFormat' in view.properties) {
    return view.properties.timeFormat
  }

  return DEFAULT_TIME_FORMAT
```

basically, rather than storing the value into a variable and returning it later, return as soon as we're certain we have a good value. otherwise return the default

Reply:
I'll test this on annotations and then make this change 👍

Reply:
I just had to tweak it a little to accommodate when timeFormat is `''`